platform/src/main/java/org/stellar/anchor/platform/component/platform/PlatformServerBeans.java

```diff
@@ -45,7 +45,15 @@ public FilterRegistrationBean<Filter> platformTokenFilter(PlatformServerConfig c
     String authSecret = config.getSecretConfig().getPlatformAuthSecret();
     switch (config.getAuth().getType()) {
       case JWT:
-        JwtService jwtService = new JwtService(null, null, null, null, null, authSecret, null);
+        JwtService jwtService =
+            new JwtService(
```

Comment:
Maybe we should start using a builder for `JwtService`. Verifying the arguments are passed in correctly without looking at the constructor is difficult.